docs/source/cli.rst

```diff
@@ -0,0 +1,56 @@
+OpenAlchemy CLI
+===============
+
+openalchemy build
+-----------------
+
+Description
+^^^^^^^^^^^
+
+Build a Python package containing the models described in the OpenAPI
+specification file.
+
+Usage
+^^^^^
+
+.. program:: openalchemy
```

Comment:
I think you just need to add `openalchemy` to the `spell.json` file under `// Library` on line 3.